File: api/api/src/main/kotlin/api/request/TestUstvariReq.kt
package api.request

import core.base.Encrypted
import core.base.Id
import core.domain.Naloga
import core.domain.Oseba
import kotlinx.datetime.LocalDate
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable

@Serializable
data class TestUstvariReq(
    @Contextual val naslov: Encrypted,
    @Contextual val podnaslov: Encrypted,
    val deadline: LocalDate,
    val oseba_ucenci_id: List<Id<Oseba>>,
    val oseba_admini_id: MutableList<Id<Oseba>>,
    val naloga_id: List<Id<Naloga>>
)

File: app/src/app/core/services/api/models/test-ustvari-req.ts
import { Naloga } from './naloga'
import { Oseba } from './oseba'
import { Id } from './id';
export interface TestUstvariReq {
	deadline: string
	naloga_id: Array<Id<Naloga>>
	naslov: string
	oseba_admini_id: Array<Id<Oseba>>
	oseba_ucenci_id: Array<Id<Oseba>>
	podnaslov: string
}